frontend/public/kubevirt/components/modals/start-stop-vm-modal.jsx

```diff
@@ -13,12 +15,36 @@ class StartStopVmModal extends PromiseComponent {
 
   submit(event) {
     event.preventDefault();
-    const patch = [{
-      op: 'replace',
-      path: '/spec/running',
-      value: this.props.start
-    }];
-    const promise = k8sPatch(this.props.kind, this.props.resource, patch);
+    let promise;
+    const vm = this.props.resource;
```

Comment:
We should start using `propTypes` and ideally even `defaultProps`. WDYT?
If you agree, do you prefer to add it within this or a follow-up PR?

Reply:
I will add propTypes within this PR